.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+
+python:
+  - "2.7"
+
```

Comment:
You also want to add a `sudo: false` line. That will make these builds run on Travis's docker infrastructure, which will be much faster.